core/src/main/java/com/oath/oak/EntrySet.java

```diff
@@ -120,7 +121,7 @@
     static final ReferenceCodec VALUE = new ReferenceCodec(32, 23, 9);
 
     private static final Unsafe unsafe = UnsafeUtils.unsafe;
-    private final MemoryManager memoryManager;
+    final MemoryManager memoryManager;
```

Comment:
Why "private" was removed?

Reply:
I removed it because I access it from `Chunk`'s constructor.